core/src/main/scala/com/salesforce/op/OpWorkflowModel.scala

```diff
@@ -419,6 +420,31 @@ class OpWorkflowModel(val uid: String = UID[OpWorkflowModel], val trainingParams
     scores -> metrics
   }
 
+  /**
+   * Creates a copy of [[OpWorkflowModel]]
+   *
+   * @return copy of [[OpWorkflowModel]]
+   */
+  def copy(): OpWorkflowModel = {
+    def copyFeatures(features: Array[OPFeature]): Array[OPFeature] = features.collect { case f: Feature[_] =>
+      implicit val tt = f.wtt
+      f.copy()
+    }
+    val copy =
+      new OpWorkflowModel(uid = uid, trainingParams = trainingParams.copy())
+        .setFeatures(copyFeatures(resultFeatures))
+        .setRawFeatures(copyFeatures(rawFeatures))
```

Comment:
you shouldnt set the raw features - they should be derived from the features

Reply:
good point. what if somebody modifies it separately? I dont want to mess up any internal state.

Reply:
they shouldnt be able to - it is private to op. setting the raw features should never be done directly. In fact the method has no usages maybe we shoudl delete it

Reply:
Ha ha ha!! Removed it.